nixos/modules/services/web-apps/doc.xml

```diff
@@ -6,7 +6,7 @@
 
   <title>Piwik</title>
   <para>
-    Piwik is a real-time web analytics application.
+    Piwik is a real-time web analytics web application.
```

Comment:
It was actually intentional, as it *is* a web application for web analytics, but I admit it sounds really weird in english, adds nothing much and probably irritates the reader more than clarifies stuff. 😆 Thanks!